.github/actions/execute-job-image/action.yml

```diff
@@ -14,10 +14,12 @@ runs:
     - name: "make push"
       shell: bash
       run: |
+        export DOCKER_BUILDKIT=1
```

Comment:
It probably makes more sense to set this globally for everything in the Makefile?